app/views/static/buttons.html.haml

```diff
@@ -1,39 +0,0 @@
-.button-group
```

Comment:
Agreed with removing this template since nobody is using it, but also remove the directive that nobody is using which uses this template please ;) (`app/assets/javascripts/directives/form_buttons.js`)